Add bloom (#1364)
iotaledger/iota-wiki: docs/get-started/wallets/bloom.md

---
title: Bloom Wallet
description: A feature-laden Shimmer wallet.
keywords:
  - Bloom Wallet
  - Shimmer Wallet
  - IOTA Wallet
  - Software Wallet
  - EVM
  - DLT
  - Cryptocurrency
---

> Bloom is a desktop wallet with first-class security developed by former members of the Firefly team. Bloom includes both Shimmer L1 and ShimmerEVM L2 in the same application, so its users can seamlessly transfer assets back and forth between Shimmer and ShimmerEVM without any outside third-party tooling. Over time, Bloom will support the entirety of the IOTA ecosystem, integrating both Shimmer and IOTA networks in a single app experience.

Reviewer: Just a suggestion to reduce the formal tone a bit and remove the pleonasm.

```suggestion
Bloom is a desktop wallet with first-class security developed by former members of the Firefly team. Bloom includes both Shimmer L1 and ShimmerEVM L2 in the same application, so you can seamlessly transfer assets back and forth between Shimmer and ShimmerEVM without the need for other tooling. Over time, Bloom will support the entirety of the IOTA ecosystem, integrating both Shimmer and IOTA networks in a single app experience.

```

Reviewer 2: Sounds good!
